Web/Admin.UI/src/app/components/tenant/acquisition-log/acquisition-log-view/acquisition-log-view.component.scss

```
@@ -99,6 +99,26 @@ $skipped-status: #ffbf00;
   font-size: 0.875rem;
 }
 
+.status-deleted {
+  @extend .log-status;
```

Stylelint (17.4.0)

[error] Expected a placeholder selector (e.g. %placeholder) to be used in `@extend` (scss/at-extend-no-missing-placeholder)

(scss/at-extend-no-missing-placeholder)